meson.build

```diff
@@ -108,6 +108,10 @@ add_project_arguments(
   c_cpp_cross_args,
   optimize_size_args,
   language: ['c', 'cpp'], native: false)
+# Add a C-only flag for cross builds.
+add_project_arguments(
+  '-Wstrict-prototypes', # Ban K&R Declarations/Definitions.
```

Comment:
Worth hoisting into a separate variable? We can leave it off and do it if more C-only flags come up.

Reply:
I didn't think it was worth it yet - there's not many warning flags that are C-only that we care about.